src/api-service/__app__/onefuzzlib/workers/nodes.py

```diff
@@ -453,7 +453,7 @@ def reimage_long_lived_nodes(cls, scaleset_id: UUID) -> None:
         reasonably up-to-date with OS patches without disrupting running
         fuzzing tasks with patch reboot cycles.
         """
-        time_filter = "Timestamp lt datetime'%s'" % (
+        time_filter = "initialized_at lt datetime'%s'" % (
```

Comment:
Since this check is already always false for older nodes, I figured older nodes would have to wait for a post-job reimage to pick up the new field. 

I will test this to see how it behaves.

Reply:
you can check [this](https://github.com/microsoft/onefuzz/blob/2ad37bdfb0be0be372940b676861b7320837b082/src/api-service/__app__/onefuzzlib/workers/nodes.py#L124) to see how we take it into account when creating the query

Reply:
Sorry Cheick, what do you mean? Are you suggesting I use 'search_outdated'?

Reply:
If we wait for older nodes to pickup jobs before upgrading and reimaging, we risk having idle nodes become really stale because of the way we statically define VMSS sizes.

Reply:
I see. I will make sure this is updated.